homeassistant/components/google_tasks/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  config-flow:
+    status: done
+    comment: |
+      The integration uses an OAuth config flow.
+  brands: done
+  dependency-transparency: done
+  common-modules:
+    status: done
+    comment: |
+      The integration has a coordinator.py and no base entities.
+  has-entity-name: done
+  action-setup:
+    status: done
+    comment: The integration does not register any actions.
+  appropriate-polling:
+    status: done
+    comment: The coordinator polls every 30 minutes.
```

Comment:
I don't like putting hard numbers in this file as it might get outdated

Reply:
Removed.